awscli/customizations/cloudformation/deploy.py

```diff
@@ -359,6 +420,41 @@ def merge_parameters(self, template_dict, parameter_overrides):
 
         return parameter_values
 
+    def _parse_input_as_json(self, arg_value):
+        # In case of inline json input it'll be list where json string
+        # will be the first element
+        if arg_value:
+            if isinstance(arg_value, str):
+                return json.loads(arg_value)
```

Comment:
Then is this the case where it got loaded from a file? If so, it might be good to include that in the comment.